src/lib/pickles/step_branch_data.ml

```diff
@@ -157,10 +157,33 @@ let create
     in
     let etyp =
       Impls.Step.input ~proofs_verified:max_proofs_verified
-        ~wrap_rounds:Backend.Tock.Rounds.n ~feature_flags
+        ~wrap_rounds:Backend.Tock.Rounds.n
       (* TODO *)
     in
-    Fix_domains.domains
+    let lookup_table_length_log2 =
+      let { Plonk_types.Features.range_check0
+          ; range_check1
+          ; foreign_field_add = _
+          ; foreign_field_mul
+          ; xor
+          ; rot
+          ; lookup
+          ; runtime_tables
+          } =
+        actual_feature_flags
+      in
+      let total_length =
+        ( if range_check0 || range_check1 || foreign_field_mul || rot then
+          Int.pow 2 12
+        else 0 )
+        + (if xor then 3 else Int.pow 2 8)
+        + (if lookup then 0 (* TODO: Need more info. *) else 0)
```

Comment:
I open [this issue](https://github.com/MinaProtocol/mina/issues/13977) to track this TODO. It is important for the use of lookup and runtime tables. I would suggest to add a link to the issue next to the TODO. This way, the reader can be redirected to a conversion outside the code if interested.